lib/pages/media-page.js

```diff
@@ -0,0 +1,50 @@
+/** @format */
+
+import { By } from 'selenium-webdriver';
+import AsyncBaseContainer from '../async-base-container';
+
+import * as driverHelper from '../driver-helper';
+import * as driverManager from '../driver-manager';
+
+export default class MediaPage extends AsyncBaseContainer {
+	constructor( driver ) {
+		super( driver, By.css( '.media-library__content' ) );
+	}
+
+	async selectRandomMedia() {
+		await this.driver.sleep( 3000 ); // TODO: try and remove this sleep
+		await driverHelper.clickWhenClickable( this.driver, By.css( '.media-library__list-item' ) );
```

Comment:
Is this selecting a random media? Or always the first media item?

Reply:
Yeah, it will always select the first image. 

@JavonDavis We already have [a function](https://github.com/Automattic/wp-e2e-tests/blob/master/lib/pages/editor-page.js#L139) which does the same, without sleep. Can you re-use it?